src/lib/Microsoft.Health.Fhir.R4.Ingest.Templates/Template/CodeValueFhirTemplateProcessor.cs

```diff
@@ -128,42 +128,42 @@ protected override Observation MergeObservationImpl(CodeValueFhirTemplate templa
             EnsureArg.IsNotNull(grp, nameof(grp));
             EnsureArg.IsNotNull(existingObservation, nameof(existingObservation));
 
-            existingObservation.Status = ObservationStatus.Amended;
+            var mergedObservation = existingObservation.DeepCopy() as Observation;

```

Comment:
Should this be using the new extension method?:

```.net
var mergedObservation = existingObservation.FullCopy()
```

Reply:
Unfortunately we can't right now.  This project is .NET 2.0 standard where as the Extenstions,Fhir.R4 project is .NET 6.0 and can't be downgraded due to dependencies.  This project needs to stay .NET Standard till we can move our RP to .NET 6.0.

I could have introduced a completely separate project just for this method but it seemed like overkill.